app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -368,8 +374,10 @@ class SettingsFragment : PreferenceFragmentCompat() {
         val preferenceOpenLinksInExternalApp =
             findPreference<Preference>(getPreferenceKey(R.string.pref_key_open_links_in_external_app))
 
-        preferencePrivateBrowsing.icon.mutate().apply {
-            setTint(requireContext().getColorFromAttr(R.attr.primaryText))
+        if (!FeatureFlags.newIconSet) {
+            preferencePrivateBrowsing.icon.mutate().apply {
+                setTint(requireContext().getColorFromAttr(R.attr.primaryText))
+            }
```

Comment:
Why do we need to use `.mutate().apply` instead of just `.apply`?

Reply:
I'm not really sure, I didn't added initially I just moved the line XD, I will investigated :)

Reply:
Thanks @cadeyrn  for the clarification :)

Reply:
Yup, otherwise any changes will affect other instances of the drawable

Reply:
Thanks 👋🏽

Reply:
Miss y'all too!